src/containerapp/azext_containerapp/_utils.py

```diff
@@ -1565,6 +1565,40 @@ def list_environment_locations(cmd):
     return res_locations
 
 
+# normalizes workload profile type
+def get_workload_profile_type(cmd, name, location):
+    return name.upper()
+
+
+def get_default_workload_profile(cmd, location):
+    return "Consumption"
+
+
+def get_default_workload_profile_from_env(cmd, env_def, resource_group):
+    location = env_def["location"]
+    api_default = get_default_workload_profile(cmd, location)
+    env_profiles = WorkloadProfileClient.list(cmd, resource_group, env_def["name"])
+    if api_default in [p["name"] for p in env_profiles]:
```

Comment:
get_default_workload_profile return "Consumption", but default workload profiles name is lower case "consumption"

```
"workloadProfiles": [
      {
        "name": "consumption",
        "workloadProfileType": "Consumption"
      }
```

Reply:
here's where rp changes to lowercase https://msazure.visualstudio.com/One/_git/AAPT-Antares-Containerapps/pullrequest/7865764?_a=files